src/js/components/ServiceForm.js

```diff
@@ -104,6 +104,40 @@ class ServiceForm extends SchemaForm {
     this.props.onChange(...arguments);
   }
 
+  validateForm() {
+    let model = this.triggerTabFormSubmit();
+
+    // Clears all previous errors.
+    FormUtil.forEachDefinition(this.multipleDefinition, function (definition) {
```

Comment:
I am probably way too late to the game here, but why are we storing `multipleDefinition` on `this`?
Couldn't we use internalStorage for this?
This is probably not something for this PR, but I thought I'd mention it, as I do not like that pattern.